website/content/en/docs/installation/install-operator-sdk.md

```diff
@@ -120,12 +120,12 @@ $ chmod +x helm-operator-${RELEASE_VERSION}-x86_64-apple-darwin && sudo mkdir -p
 $ git clone https://github.com/operator-framework/operator-sdk
 $ cd operator-sdk
 $ git checkout master
-$ make tidy
+$ make build
```

Comment:
Good catch on `make tidy` being removed. I missed this in #4023. Tidying `go.mod` should no longer be necessary in these instructions since we enforce `go mod tidy` to be run for every PR.

I think we can just remove this line entirely.

```suggestion
```